docs/developer/typescript/01_gettingStarted/01_hello/01_ecs.md

```diff
@@ -0,0 +1,86 @@
+# The ECS Pattern
+The [Entity Component System](https://en.wikipedia.org/wiki/Entity_component_system) is a pattern used to organize our code when writing software.  
+In this pattern:
+- Logic is represented as `System`s, and they define the behavior of the application.
+- Data is represented as `Component`s that have no behavior or identifiers attached to them.
+- Components are attached to Entities.
+- An `Entity` is an identifier.  
+  Each entity is essentially a "name" that groups components into a single "thing" (an object).
```

Comment:
![image](https://github.com/EtherealEngine/etherealengine-docs/assets/7308253/ad94dece-7d92-4d12-84c7-9d616e1193eb)
![image](https://github.com/EtherealEngine/etherealengine-docs/assets/7308253/04c9f2a5-660f-4d9e-9b43-5425ae974e23)
The sentence is capturing what an Entity "is" not what it is "used for".
But you are going beyond my knowledge of etymology, so please write your proposed sentence and it will be pushed as is.